ingesters/HttpIngester/config.go

```diff
@@ -66,6 +67,7 @@ type lst struct {
 	Assume_Local_Timezone     bool
 	Timezone_Override         string
 	Timestamp_Format_Override string //override the timestamp format
+	Timestamp_Cutoff          string
```

Comment:
this isn't documented yet, I think this should be a duration NOT a timestamp.

The timestamp will just mean you have to continually update it or pick something that won't ever exactly fit.

I would rather say nothing older than -2 years or +2d to control outright bad timestamps making it in.